> How exactly error looks like?

```
FAILED: [code=1] compiler-rt/lib/asan_abi/CMakeFiles/RTASAN_ABI.iossim.dir/asan_abi_shim.cpp.o
/Users/aidengrossman/programming/llvm-project/build/bin/clang++ --target=arm64-apple-darwin24.6.0 -D_DEBUG -D_GLIBCXX_ASSERTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/Users/aidengrossman/programming/llvm-project/compiler-rt/lib/asan_abi/.. -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wno-pass-failed -Wmisleading-indentation -Wctad-maybe-unsupported -fdiagnostics-color -Wall -Wno-unused-parameter -O3 -DNDEBUG -std=c++17 -arch arm64 -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -stdlib=libc++ -mios-simulator-version-min=9.0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.5.sdk -fPIC -fno-builtin -fno-exceptions -funwind-tables -fno-stack-protector -fno-sanitize=safe-stack -fvisibility=hidden -fno-lto -Wthread-safety -Wthread-safety-reference -Wthread-safety-beta -O3 -g -Wno-gnu -Wno-variadic-macros -Wno-c99-extensions -ftrivial-auto-var-init=pattern -nostdinc++ -MD -MT compiler-rt/lib/asan_abi/CMakeFiles/RTASAN_ABI.iossim.dir/asan_abi_shim.cpp.o -MF compiler-rt/lib/asan_abi/CMakeFiles/RTASAN_ABI.iossim.dir/asan_abi_shim.cpp.o.d -o compiler-rt/lib/asan_abi/CMakeFiles/RTASAN_ABI.iossim.dir/asan_abi_shim.cpp.o -c /Users/aidengrossman/programming/llvm-project/compiler-rt/lib/asan_abi/asan_abi_shim.cpp
/Users/aidengrossman/programming/llvm-project/compiler-rt/lib/asan_abi/asan_abi_shim.cpp:369:3: error: void function '__asan_suppress_fake_stack' should not return a value [-Wreturn-mismatch]
  369 |   return __asan_abi_suppress_fake_stack();
      |   ^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/aidengrossman/programming/llvm-project/compiler-rt/lib/asan_abi/asan_abi_shim.cpp:372:3: error: void function '__asan_unsuppress_fake_stack' should not return a value [-Wreturn-mismatch]
  372 |   return __asan_abi_unsuppress_fake_stack();
      |   ^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2 errors generated.
```

--- a/compiler-rt/lib/asan_abi/asan_abi.h
+++ b/compiler-rt/lib/asan_abi/asan_abi.h
@@ -76,8 +76,8 @@ void *__asan_abi_load_cxx_array_cookie(void **p);
 void *__asan_abi_get_current_fake_stack();
 void *__asan_abi_addr_is_in_fake_stack(void *fake_stack, void *addr, void **beg,
                                        void **end);
-void *__asan_abi_suppress_fake_stack();
-void *__asan_abi_unsuppress_fake_stack();
+void __asan_abi_suppress_fake_stack();
+void __asan_abi_unsuppress_fake_stack();
 
 // Functions concerning poisoning and unpoisoning fake stack alloca
 void __asan_abi_alloca_poison(void *addr, size_t size);

--- a/compiler-rt/lib/asan_abi/asan_abi_shim.cpp
+++ b/compiler-rt/lib/asan_abi/asan_abi_shim.cpp
@@ -365,12 +365,8 @@ void *__asan_addr_is_in_fake_stack(void *fake_stack, void *addr, void **beg,
                                    void **end) {
   return __asan_abi_addr_is_in_fake_stack(fake_stack, addr, beg, end);
 }
-void __asan_suppress_fake_stack(void) {
-  __asan_abi_suppress_fake_stack();
-}
-void __asan_unsuppress_fake_stack(void) {
-  __asan_abi_unsuppress_fake_stack();
-}
+void __asan_suppress_fake_stack(void) { __asan_abi_suppress_fake_stack(); }
+void __asan_unsuppress_fake_stack(void) { __asan_abi_unsuppress_fake_stack(); }
 
 // Functions concerning poisoning and unpoisoning fake stack alloca
 void __asan_alloca_poison(uptr addr, uptr size) {

